### PR TITLE
include multiple elements and two types of materials

### PR DESCRIPTION
**Pull request by mvanzulli**
This example aims to capture the dynamic between truss and beam element interacction. Nowdays is executed only with truss nodal gravity forces and conssitent mass matrix. Results are:
Uz end frame node:
uz_frame
Uz end turss node:
uz_truss

### CODE REVIEW

File: examples/beamTrussPendulum/onsasExample_beamTrussPendulum.m [hunk 1]
```diff
@@ -28,12 +28,12 @@
 %md
 %mdTruss:
 elements(2).elemType = 'truss'          ;
-elements(2).elemTypeGeometry = [ 3 dt ] ;
+elements(2).elemTypeGeometry = [ 3 dt dt ] ;
```
Comment (jorgepz) [inline examples/beamTrussPendulum/onsasExample_beamTrussPendulum.m:31]: no entiendo porque le ponés dos parámetros a la sección...

Comment (mvanzulli) [reply examples/beamTrussPendulum/onsasExample_beamTrussPendulum.m]: Modified

**Comment by mvanzulli** [comment]
This example now is not working until circular section vtk is implemented into master. However was useful to detect the bug.

**Review by jorgepz — APPROVED** [review]

**Comment by mvanzulli** [comment]
The plot is still not working but this PR is completed

**Pull request merged**